### PR TITLE
Pass account id to queries

### PR DESCRIPTION
**Pull request by mre**
In `v1` the bearer key (account id) must be equal to the podcast id.
in later versions this no longer has to be the case. This way we can have multiple podcast ids per account id in the future.

### CODE REVIEW

File: db_schema/queries/v1/episodesAge.sql [hunk 1]
```diff
@@ -4,6 +4,7 @@ WITH data as
   FROM spotifyEpisodeAggregate
   WHERE
   spa_facet_type="age"
+  AND account_id = @podcast_id 
   AND spa_date >= @start
   AND spa_date <= @end
   AND account_id = 1
```
Comment (woolfg) [inline db_schema/queries/v1/episodesAge.sql:10]: has to be removed (in all sql files)

File: src/index.ts [hunk 2]
```diff
@@ -188,32 +188,41 @@ app.get(
...
 // The endpoint must contain a version number and a query name
-// e.g. /analytics/v1/someQuery
+// e.g. /analytics/v1/1234/someQuery
+// where 1234 is the podcast id and someQuery is the name of the query
 app.get(
-    '/analytics/:version/:query',
+    '/analytics/:version/:podcastId/:query',
     async (req: Request, res: Response, next: NextFunction) => {
         try {
+            const accountId = res.locals.user.accountId
```
Comment (woolfg) [inline src/index.ts:197]: just to be on the safe side, we should check if account id or podcast id is not set and throw an error

File: tests/api_e2e/analytics.test.js [hunk 3]
```diff
@@ -28,16 +30,21 @@ describe('check basic analytics query', () => {
         expect(response.statusCode).toBe(401)
     })
 
+    it('should return status 401 when sending query without podcast id', async () => {
```
Comment (woolfg) [inline tests/api_e2e/analytics.test.js:33]: maybe we can add a check if podcast id != account id

**Review by woolfg — CHANGES_REQUESTED** [review]
- the @var approach is cool, was just a small change :+1: 
- sql files: account_id=1 has to be removed
- would add some security checks (see comments)

File: tests/api_e2e/analytics.test.js (hunk 3 above)
Comment (mre) [reply tests/api_e2e/analytics.test.js:44]: good idea; added

**Comment by mre** [comment]
> sql files: account_id=1 has to be removed

Sorry, that was because of a bad merge. It's fixed now I hope.

**Review by woolfg — APPROVED** [review]
lgtm

**Pull request merged**